filters/SkewnessBalancingFilter.cpp

```diff
@@ -67,7 +67,17 @@ std::set<PointId> SkewnessBalancingFilter::processGround(PointViewPtr view)
     point_count_t n1(0);
     double delta, delta_n, term1, M1, M2, M3;
     M1 = M2 = M3 = 0.0;
-    std::vector<double> skewness(view->size());
+
+    auto setGround = [&view](PointId start, PointId end)
+    {
+        // 2 is the ASPRS LAS classification for ground.
+        for (PointId idx = start; idx <= end; ++idx)
+            view->setField(Dimension::Id::Classification, idx, 2);
```

Comment:
This isn't so much a comment on what you've done here, but it dawns on me that we should probably change this filter to behave more like the other ground filters, setting those points that are not ground to a Classification value of 1. Would clearly happen elsewhere in the code though.

Reply:
I can do that here.  Do you want me to do that?

Reply:
I think we should. Thanks.